sources/packages/backend/libs/integrations/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -40,136 +43,140 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         applicationStatus: Not(ApplicationStatus.Edited),
       },
```

Comment:
Will it give the same outcome if the application status and offering intensity criteria can move outside the assessment base criteria. ?

<img width="568" height="165" alt="Image" src="https://github.com/user-attachments/assets/4c795491-6b63-4f91-b0d8-bd1f62a55df0" />

Reply:
It should be, I will move them.

Reply:
Move to the `where`. A minor adjustment to the IER12 factory was required.